custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -337,6 +344,92 @@ def handle_exception(self, exception, repeat_record):
                                 "followup_nikshay_registered", "followup_nikshay_error")
 
 
+@RegisterGenerator(NikshayRegisterPrivatePatientRepeater, 'case_xml', 'XML', is_default=True)
```

Comment:
I think this is very misleading, since this doesn't actually return XML. Is there a reason you wouldn't make a new `SOAP` type like I had in my WIP?